src/css/components/header-buttons-mobile.css

```diff
@@ -1,3 +1,16 @@
+.js-overlay-active .mobile-buttons {
+  position: fixed;
+  top: 1rem;
+  right: 1rem;
+  z-index: 10;
```

Comment:
Add a comment on a the `z-index` indicating what it's intended to appear above..